### PR TITLE
Fixed #14808 -- Doc'd that trans and blocktrans tags don't escape translations.

### PR DESCRIPTION
**Pull request by rixx**
https://code.djangoproject.com/ticket/14808

### CODE REVIEW

File: docs/topics/i18n/translation.txt [hunk 1]
```diff
@@ -567,6 +567,13 @@ As with all template tags, this tag needs to be loaded in all templates which
 use translations, even those templates that extend from other templates which
 have already loaded the ``i18n`` tag.
 
+.. note::
```
Comment (felixxm) [inline docs/topics/i18n/translation.txt]: I think we can use `.. warning` instead of `.. note`.

File: docs/topics/i18n/translation.txt [hunk 2]
```diff
@@ -567,6 +567,13 @@ As with all template tags, this tag needs to be loaded in all templates which
...
+
+    Internationalized strings will not be escaped when rendered in a template.
```
Comment (felixxm) [inline docs/topics/i18n/translation.txt]: Maybe `Internationalized` -> `Translated`?

File: docs/topics/i18n/translation.txt [hunk 3]
```diff
@@ -567,6 +567,13 @@ As with all template tags, this tag needs to be loaded in all templates which
...
+    This allows you to include HTML in translations, for example for emphasis,
```
Comment (felixxm) [inline docs/topics/i18n/translation.txt]: Should we put comma before `for emphasis`? :thinking:

File: docs/topics/i18n/translation.txt [hunk 4]
```diff
@@ -567,6 +567,13 @@ As with all template tags, this tag needs to be loaded in all templates which
...
+    but it also means that potentially dangerous characters (e.g. ``"``) will
+    be rendered unchanged.
```
Comment (felixxm) [inline docs/topics/i18n/translation.txt]: I'm not sure if we need `it also means that ` :thinking: . Maybe:
```
but potentially dangerous characters (e.g. ``"``) will also be rendered unchanged.
```

File: docs/topics/i18n/translation.txt (hunk 3 above)
Comment (rixx) [reply docs/topics/i18n/translation.txt]: Not sure, feels weird to me, but I'm not a native speaker.

**Comment by rixx** [comment]
Thank you for the review – pushed the changes you requested, except for the comma, which I'm not sure about.

File: docs/topics/i18n/translation.txt (hunk 3 above)
Comment (felixxm) [reply docs/topics/i18n/translation.txt]: As you know me neither :man_shrugging:  :smile:

**Pull request merged**